### PR TITLE
add mdbenchmark

### PR DESCRIPTION
**Pull request by kain88-de**
a small utility to benchmark MD simulations on HPC clusters.

### CODE REVIEW

File: recipes/mdbenchmark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - matplotlib
+    - xdg <2
+    - python-levenshtein
+
+test:
+  imports:
+    - mdbenchmark
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (synapticarbors) [inline recipes/mdbenchmark/meta.yaml]: Incorrect url

File: recipes/mdbenchmark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  license: GPL
+  license_family: GPL3
```
Comment (synapticarbors) [inline recipes/mdbenchmark/meta.yaml]: Should be `license: GPL-3.0` and `license_family: GPL`

File: recipes/mdbenchmark/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
```
Comment (synapticarbors) [inline recipes/mdbenchmark/meta.yaml]: Has an entry_point:

https://github.com/bio-phys/MDBenchmark/blob/master/setup.py#L72

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/mdbenchmark/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - numpy >=1.8
+    - mdsynthesis
+    - click
+    - jinja2
+    - pandas
+    - matplotlib
+    - xdg <2
+    - python-levenshtein
+
+test:
```
Comment (synapticarbors) [inline recipes/mdbenchmark/meta.yaml]: Test the command line tool:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

**Comment by kain88-de** [comment]
> It's failing on Travis due to a missing file which I think should be clear from the error message.

yeah I noticed. My way to guess the number of cores isn't working on osx. So for now this is a unix only package.

Thanks for the review.

File: recipes/mdbenchmark/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    - mdbenchmark = mdbenchmark.cli:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.8
+    - mdsynthesis
```
Comment (synapticarbors) [inline recipes/mdbenchmark/meta.yaml]: It looks like `mdsynthesis` is py27 only:

https://github.com/conda-forge/mdsynthesis-feedstock/blob/master/recipe/meta.yaml#L16

Comment (kain88-de) [reply recipes/mdbenchmark/meta.yaml]: Oh yeah right. I only tested the package on my own laptop which already has python3 preview builds for everything. This has to wait then until the dependencies are updated.

**Comment by kain88-de** [comment]
build has been fixed.

**Pull request merged**